packages/landing/src/config.map.ts

```diff
@@ -107,9 +109,12 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.style = style ?? null;
     this.layerId = layerId.startsWith('im_') ? layerId.slice(3) : layerId;
     this.tileMatrix = tileMatrix;
+    this.dateRange.dateBefore = dateBefore ?? undefined;
+    this.dateRange.dateAfter = dateAfter ?? undefined;
```

Comment:
do we want to validate that these dates are valid? `new Date().getTime()` and dateBefore < dateAfter?

Reply:
Good point